LandLord-core/src/main/java/biz/princeps/lib/storage/Datastorage.java

```diff
@@ -181,24 +151,22 @@ public Triplet executeQuery(String query, Object... args) {
     }
 
     private void evalutePrepStmt(PreparedStatement st, Object... args) throws SQLException {
-        if (args != null) {
-            for (int i = 0; i < args.length; i++) {
-                Object obj = args[i];
-                if (obj instanceof String)
-                    st.setString(i + 1, (String) obj);
-                else if (obj instanceof Integer)
-                    st.setInt(i + 1, (int) obj);
-                else if (obj instanceof Double)
-                    st.setDouble(i + 1, (double) obj);
-                else if (obj instanceof Float)
-                    st.setFloat(i + 1, (float) obj);
-                else if (obj instanceof Boolean)
-                    st.setBoolean(i + 1, (boolean) obj);
-                else if (obj instanceof Long)
-                    st.setLong(i + 1, (long) obj);
-                else
-                    st.setNull(i + 1, Types.VARCHAR);
-            }
+        for (int i = 0; i < args.length; i++) {
```

Comment:
Always thought that this is the hackiest stuff I ever saw xD
Also it breaks with a stuff from the time api and some other special cases.
Nothing to change now, but maybe we should clean up the queries and use direct typing and not this verbose type detection.

Reply:
Sql is still unsupported tho, just did that for future refactoring. Let’s use your library.